src/analyze/analyze.c

```diff
@@ -227,24 +214,24 @@ static void subtract_timestamp(usec_t *a, usec_t b) {
 
 static int acquire_boot_times(sd_bus *bus, struct boot_times **bt) {
         static const struct bus_properties_map property_map[] = {
-                { "FirmwareTimestampMonotonic",               "t", NULL, offsetof(struct boot_times, firmware_time)                 },
```

Comment:
These "custom alignments" get rendered away with clang-format. 
Possible Workarounds, if required:
- disabling clang-format for code range
- align by adding aliging comments, eg: "LoaderTimestampMonotonic" /*********/, "t", ...